### ISSUE TITLE
The default fhir-server-config file-based bulk storageProvider should point to an existing directory

### ISSUE THREAD

**Issue by lmsurpre** [IBM/FHIR#2086]
**Is your feature request related to a problem? Please describe.**
Our default config has `"fileBase": "/fhir-server/output"`.  However, this location doesn't exist by default and so, to make it functional, a user must either:
A. change this default setting in fhir-server-config.json; or
B. create this absolute directory for the output

I believe that this was done so that our config would work in our CI across both windows and linux.

**Describe the solution you'd like**
Our docker container already has a symlink from `/output` to `/opt/ol/wlp/usr/servers/fhir-server` (with the proper user permissions I think), so maybe we can just create a directory name `bulkdata` in that location and set the default to `/output/bulkdata`?

**Describe alternatives you've considered**
Document steps for having a volume mount at the existing default directory instead:  `/fhir-server/output`

**Acceptance Criteria**
1.  
   GIVEN [a precondition]
    AND [another precondition]
   WHEN [test step]
    AND [test step]
   THEN [verification step]
    AND [verification step]

**Additional context**
Add the target directory to our docker image

**Comment by lmsurpre** [comment]
Paul recommended that we include an extra level to make it easier to do mounts properly.
For example, set the config to default to `/output/bulkdata/data` instead of `/output/bulkdata`.  This enables users to mount `/output/bulkdata`.

**Comment by lmsurpre** [comment]
I built the latest docker image and then ran it with a command like this:
> docker run -it -p 9443:9443 -v /tmp/derby:/output/derby -e BOOTSTRAP_DB=true ibmcom/ibm-fhir-server

I loaded a single patient and then invoked export.  To my delight, it "just works" with the default config now. The exported file was available within the container under /output/bulkdata.

Next I added an additional volume mount as follows:
> docker run -it -p 9443:9443 -v /tmp/derby:/output/derby -v /tmp/bulkdata:/output/bulkdata ibmcom/ibm-fhir-server

I repeated the exported and now the export was under my host's /tmp/bulkdata directory.  Awesome.
